pkg/translate/terraform_provider/template.go

```diff
@@ -1544,6 +1544,16 @@ func (p *PanosProvider) Configure(ctx context.Context, req provider.ConfigureReq
 		}
 	} else {
 		tflog.Info(ctx, "Configuring client for API mode")
+		var logCategories sdk.LogCategory
+		if !config.SdkLogCategories.IsNull() {
+			categories := strings.Split(config.SdkLogCategories.ValueString(), ",")
+			var err error
```

Comment:
question: Can we assign the error value directly to the variable without defining using `var`?

Reply:
We can't use golang declaration + assignment here (:=) as logCategories would be shadowed by the new variable declared inside of the if block.